pkg/GAPJulia/JuliaInterface/gap/adapter.gi

```diff
@@ -55,19 +55,39 @@ InstallOtherMethod( \[\]\:\=,
 #
 # "matrices" / lists of lists
 #
+#TODO:  Remove the installations for '\[\]' and '\[\]\:\='
+#       as soon as a released GAP version supports '\[\,\]' and '\[\,\]\:\='.
+if IsBound( \[\,\] ) then
+InstallOtherMethod( \[\,\],
```

Comment:
Can't we just install methods for MatElm and SetMatElm? At least that was the theory, that this way one could write the code so that it works with 4.9, 4.10 and 4.11

Reply:
Argh, no of course not, and you already explained the reason in your PR description. Sorry about that